TROGDOR.md

```diff
@@ -143,7 +143,7 @@ RoundTripWorkload tests both production and consumption.  The workload starts a
 
 ### ConsumeBench
 ConsumeBench starts one or more Kafka consumers on a single agent node. Depending on the passed in configuration (see ConsumeBenchSpec), the consumers either subscribe to a set of topics (leveraging consumer group functionality and dynamic partition assignment) or manually assign partitions to themselves.
-The workload measures the average produce latency, as well as the median, 95th percentile, and 99th percentile latency.
+The workload measures the average consume latency, as well as the median, 95th percentile, and 99th percentile latency.
```

Comment:
Have we verified by running the test that it actually doesn't measure produce latency?

Reply:
> Have we verified by running the test that it actually doesn't measure produce latency?

No, but I read the code of org.apache.kafka.trogdor.workload.ConsumeBenchWorker.java and confirmed that the calculation of latency is related to consume rather than produce. @divijvaidya

Reply:
![WechatIMG30](https://github.com/apache/kafka/assets/16536770/5e0cdd96-8c86-4016-a452-700975c1317f)

@divijvaidya Hi, I have run the test, it is used to measure consume latency.